xbmc/guilib/GUIEditControl.cpp

```diff
@@ -747,3 +747,11 @@ void CGUIEditControl::SetFocus(bool focus)
   g_Windowing.EnableTextInput(focus);
   CGUIControl::SetFocus(focus);
 }
+
+std::string CGUIEditControl::GetDescriptionByIndex(int index) const
+{
+  if (index == 0)
+    return GetDescription();
+
+  return GetLabel2();
```

Comment:
no this looks still odd to me - that index is bascially not an index but a flag? ehrm ;)